initfiles/bin/init_dafilesrv.in

```diff
@@ -44,8 +44,8 @@ killed(){
     kill_process ${SENTINEL} ${PID_NAME} 3
     exit 255
 }
-ulimit -c unlimited
-ulimit -n $handlelimit
+
+
 trap "killed" SIGINT SIGTERM SIGKILL
 dafilesrv -L $log -I ${INSTANCE_NAME} &
```

Comment:
service dafilesrv start -> /etc/init.d/dafilesrv -> (hpcc_common) start_dafilesrv() -> (hpcc_common) startCmd()

It will hit our logic.